### DISCUSSION TITLE
Thrustmaster force feedback gt

### DISCUSSION THREAD

**Discussion by survivor182ca** [berarma/oversteer#115] [Ideas]
Hi,

I own this wheel and I was wondering if you could add support for that wheel

The wheel is recognized in kernel 5.15 and it does work properly (without force feedback) with qjoypad

https://support.thrustmaster.com/en/product/ffdrw-en/

Im using linuxmint 21 and oversteer doe not recognized my wheel.

Thank in advance.

**Comment by berarma** [comment]
Yes, I need the info from the lsusb command.
